deps/3rdparty/libdispatch/build/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "WinObjC.Language": "*",
```

Comment:
Why? I don't see you adding code that uses ObjC?

Reply:
LibDispatch must be compiled with ClangCompile though. This exists in the WinObjC Language package.